packages/react-instantsearch/src/connectors/connectRefinementList.js

```diff
@@ -100,9 +100,23 @@ export default createConnector({
         isRefined: v.isRefined,
       }));
 
+    let facetValues;
+    if (facetValuesResults) {
+      facetValues = facetValuesResults[attributeName]
+        .slice(0, limit).map(
+          v => ({
+            label: v.value,
+            value: getValue(v.value, props, searchState),
+            _highlightResult: {label: {value: v.highlighted}},
+            count: v.count,
+            isRefined: v.isRefined,
+          }));
+    }
+
     return {
       items,
       currentRefinement: getCurrentRefinement(props, searchState),
+      facetValues,
```

Comment:
I like searchedFacetValues but maybe itemsFromSearch is closer than what we already have.